policyengine_us/parameters/gov/states/ct/tax/income/subtractions/magi_excess.yaml

```diff
@@ -0,0 +1,13 @@
+description: Connecticut calculates this fraction of the modified adjusted gross income excess, under the social security benefit adjustment.
+values:
+   2019-01-01: 0.25
+metadata:
+  unit: /1
+  period: year
+  label: Connecticut modified adjusted gross income excess rate
+  reference:
+    # The Connecticut tax form does not mention this rate
```

Comment:
Yes it does - on page 24 as described on page 8

In general though only cite sources that corroborate the parameter values. If the form didn't mention it then we should not include a reference to it.